act-rules-format.bs

```diff
@@ -633,27 +646,28 @@ This section is *non-normative*.
 }
 ```
 
-**Example 3:** Aggregate outcomes to a WCAG success criterion
+**Example 3:** Aggregating based on 'Requirement' (eg. WCAG Success Criteria)
 
 ```javascript
 {
-  "@context": "https://raw.githubusercontent.com/w3c/wcag-act/master/earl-act.json",
+  "@context": "context.json",
   "@type": "Assertion",
+  "assertor": "MyTool",
   "subject": {
     "@type": ["WebPage", "TestSubject"],
     "@value": "https://example.org/page1.html"
   }
-  "test": "wcag21:captions-prerecorded",
+  "test": "WCAG21:captions-prerecorded",
   "result": {
     "outcome": "earl:failed",
     "source": [{
-      "test": "auto-wcag:media-equiv-av-only-alt",
+      "test": "ACT-CG:rules/f51b46",
       "result": {
         "outcome": "earl:failed",
         "source": [ … ]
```

Comment:
It doesn't really make sense for this to have another `source` property. Let's take this out.

Reply:
Done.